client/devdocs/docs-example/style.scss

```diff
@@ -62,4 +62,10 @@
 			height: 50px;
 		}
 	}
+
+	.gridicons-link {
+		margin-left: 6px;
```

Comment:
Would padding be suitable?

Reply:
Hmm, good question. I've never really explicitly come up with a rule for when to use which. 

After some searching, it seems like `margin` might be better in this case, since [it auto-collapses with adjacent margins](https://stackoverflow.com/a/9183818/450127) and the purpose here is to [separate it from another element](https://stackoverflow.com/a/2189462/450127), rather than from it's own border.

What do you think? Do you see a reason why `padding` might fit better? They seem to behave identically in this case, from my brief testing.